frontend/src/metabase/actions/components/ActionFormFieldWidget/ActionFormFieldWidget.tsx

```diff
@@ -32,16 +33,24 @@ interface FormWidgetProps {
   formField: ActionFormFieldProps;
 }
 
-const ActionFormFieldWidget = forwardRef(function FormFieldWidget(
+export const ActionFormFieldWidget = forwardRef(function FormFieldWidget(
   { formField }: FormWidgetProps,
   ref: React.Ref<any>,
 ) {
+  const [hidden, setHidden] = useState(false);
```

Comment:
Mapping this to the correct property will be done in another PR?

Reply:
I pushed it by mistake, sorry